changelog.html

```diff
@@ -55,8 +55,52 @@
 
 <!-- Record your changes in the trunk here. -->
 <div id="trunk" style="display:none"><!--=TRUNK-BEGIN=-->
+
+<div style="margin: 10px; padding: 10px; background-color: #FFFFCE;">
+  <strong>More detailed information about the new features in Jenkins 2.0 <a href="/2.0/">on the overview page</a>.</strong>
+</div>
+
 <ul class=image>
-  <li class=>
+  <li class="rfe major">
+    New password-protected setup wizard shown on first run to guide users through installation of popular plugins and setting up an admin user.
+    (<a href="https://issues.jenkins-ci.org/browse/JENKINS-30749">issue 30749</a>,
+     <a href="https://issues.jenkins-ci.org/browse/JENKINS-9598">issue 9598</a>)
+  <li class="rfe major">
+    Plugin bundling overhaul: Bundled plugins are only installed if necessary when upgrading, all plugins can be uninstalled.
+    (<a href="https://issues.jenkins-ci.org/browse/JENKINS-20617">issue 20617</a>)
+  <li class="rfe major">
+    Redesigned job configuration form makes it easier to see the option hierarchy, and to navigate the form using scrolling tabs.
```

Comment:
@jtnord If you know a better name than *scrolling tabs* to describe them, I'd like to know.